circle.yml

```diff
@@ -18,7 +18,7 @@ machine:
 
 dependencies:
   pre:
-    - wget https://github.com/harvard-lts/fits/archive/1.0.5.zip -O fits-1.0.5.zip
+    - wget http://projects.iq.harvard.edu/files/fits/files/fits-1.0.5.zip -O fits-1.0.5.zip
```

Comment:
Why the -O when wget by default saves to the filename?

Reply:
Because someone copy/pasted that from someone else's `circle.yml` file I'm guessing ;) Good call though. I'll drop it.